pyproject.toml

```
@@ -42,7 +42,10 @@ Homepage = "https://github.com/ASFHyP3/hyp3-ISCE2"
 Documentation = "https://hyp3-docs.asf.alaska.edu"
 
 [project.scripts]
-topsapp_burst = "hyp3_isce2.process:main"
+insar_tops_burst = "hyp3_isce2.insar_tops_burst:main"
```

Overall, I still don't really understand the motivation for this entry point -- it looks like it's just o make it so you can run:
```shell
insar_tops_burst [ARGS]
```
instead of 
```shell
python -m hyp3_isce2 [ARGS]
```
as they only differ by the `--bucket` and `--bucket-prefix` arguments, which *are* optional. 

If it's a convenience thing (not wanting to type `python -m ` in front) then I'd ranter see a script entrypoint like:
```suggestion
hyp3_isce2 = "hyp3_isce2.__main__:main"
```
you you'd be able to just run:
```shell
hyp3_isce2 [ARGS]
```

**If** it's for actually different behavior anticipated in the future like some of our other "science" code, I can see adding an extra script entrypoint for convenience, but I'd rather do it *then*.

---

:thinking: I suppose, for the non-default HyP3 entry point, you'll need to add `++process [PROC_NAME]`  which does get a bit long:
```
python -m hyp3_isce2 ++process [PROC_NAME] [ARGS]
# Or
hyp3_isce2 ++process [PROC_NAME] [ARGS]
```
so I guess having specific workflow entry points *might* make sense. In that case, 

I think I'd rather just see the `[project.scripts]` and `[project.entry-points.hyp3]` sections of `pyproject.toml` be exactly the same, or potentially better, not using the argparse trick at all and building out a proper CLI with sub parsers:
```shell
hyp3_isce2 [PROC_NAME] [ARGS]
```